hathorlib/base_transaction.py

```diff
@@ -393,6 +396,24 @@ def is_nft_creation(self) -> bool:
         """Returns True if it's an NFT creation transaction"""
         return False
 
+    def is_standard(self, max_output_script_size: int = settings.MAX_OUTPUT_SCRIPT_SIZE,
+                    allow_non_standard_script: bool = False) -> bool:
```

Comment:
I'm a little confused by the `allow_non_standard_script` param. 

My understanding was that if allow non-standard scripts, this method would just not be called, that is `tx.is_standard(allow_non_standard_script=True)` would always return `True` and it would be preferable to let the caller choose whether to call the method instead.

Reply:
The idea is that a standard transaction is validated by different checks. If `allow_non_standard_script = True`, `is_standard` can still be `False` if the size is bigger than the maximum allowed.

This parameter refers specifically to the output script and there are other things in the transaction that could make it non standard.

Does it make sense?

Reply:
It was fixed in the last commit.